CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 4.9.0
+- fix: Ensure context from previous run is used for native main crashes (#683)
+- feat: Tracing without performance (#710)
+- feat: Deprecate `tracingOrigins` for `tracePropagationTargets` (#703)
+- feat: Use `protocol.handle` on Electron >= v25 (#704)
+- test: Update Sentry SDKs to v7.60.0 (#706)
```

Comment:
would nice nice to

a) say what versions we moved from (`v7.58.0` to `v7.60.0`)
b) link the release like we did below for `4.8.0`.